.azure_pipelines/olive-ci.yaml

```diff
@@ -150,6 +150,10 @@ jobs:
         exampleFolder: stable_diffusion
         exampleName: stable_diffusion_cuda_gpu
         exampleRequirements: requirements-common.txt
+      llama2:
```

Comment:
do you know how long this might take to run? from my experience (on a machine with big CPU), the transformers optimization still takes a fairly long time.

Reply:
I thought the idea was to have a large model pipeline that runs less regularly?

Reply:
No I don't know. I was trying to test this in this PR. If it takes long time, I will add this to another pipeline.

Reply:
I see. Makes sense. Thanks!

Reply:
llama2 test takes 26m while our unit test takes 30m. It looks good to me.